plugins/telemetry/telemetry.h

```diff
@@ -6,48 +6,90 @@ namespace dronecore {
 
 class TelemetryImpl;
 
+/**
+ * The Telemetry class allows to get all kinds of telemetry infos from a device.
```

Comment:
I'd say a bit more: - 

The Telemetry class allows users to get vehicle telemetry and state information (e.g. battery, GPS, RC connection, flight mode etc.) and set telemetry update rates.

Reply:
Thx.